pkg/kubelet/dockertools/manager.go

```diff
@@ -1494,14 +1494,28 @@ func (dm *DockerManager) runContainerInPod(pod *api.Pod, container *api.Containe
 	if err != nil {
 		return kubecontainer.ContainerID{}, fmt.Errorf("InspectContainer: %v", err)
 	}
+
+	if !containerInfo.State.Running {
```

Comment:
Without this fix, we couldn't even get the pid because the container already exited and the function just errors out immediately.

Another option is to not check the pid here at all, and let `applyOOMScoreAdjIfneeded` to handle pid == 0. What do you think?